extmod/moductypes.c

```diff
@@ -375,7 +375,7 @@ STATIC void set_aligned(uint val_type, void *p, mp_int_t index, mp_obj_t val) {
     if (val_type == FLOAT32 || val_type == FLOAT64) {
         mp_float_t v = mp_obj_get_float(val);
         if (val_type == FLOAT32) {
-            ((float *)p)[index] = v;
+            ((float *)p)[index] = (float) v;
```

Comment:
Does uncrustify still not remove the space in the cast here?

Reply:
It does AFAIK but this commit is from before changing he configuration. Also everything still needs rebasing.